source/cbmtext.cpp

```diff
@@ -52,7 +52,7 @@ void printText(ostream& out, istream& in) {
 
 char* translate(uint32_t c, char *code) {
 	if (isSpace(c)) c = 32;
-	if (c >= 32 && c <= 127) {
+	if (c !=34 && c >= 32 && c <= 127) { // print doublequote as symbol
```

Comment:
There is no need for this exception for chr(34): the definition will be found in the structure and will be used instead of single char, even without modifying the IF statement.

Reply:
Thanks! I'm not very experienced in C++.